crates/ruma-signatures/Cargo.toml

```
@@ -13,8 +13,12 @@ version = "0.7.1"
 
 [dependencies]
 base64 = "0.13.0"
-ring = "0.16.19"
+ed25519-dalek = "1.0.1"
+pkcs8 = { git = "https://github.com/RustCrypto/utils", revision = "51e7c9d734e4d3c5279ba1c181c65b1bd77bcad0", features = ["alloc"] }
```

I have never seen `revision` before and only find `rev` in the Cargo docs.

```suggestion
pkcs8 = { git = "https://github.com/RustCrypto/utils", rev = "51e7c9d734e4d3c5279ba1c181c65b1bd77bcad0", features = ["alloc"] }
```

---

`revision` works as well, but alright